physics/smoke_dust/module_add_emiss_burn.F90

```diff
@@ -55,12 +55,15 @@ subroutine add_emis_burn(dtstep,dz8w,rho_phy,pi,ebb_min,          &
    real(kind_phys) :: timeq, fire_age, age_hr, dt1,dt2,dtm         ! For BB emis. diurnal cycle calculation
 
 ! For Gaussian diurnal cycle
-   real(kind_phys), PARAMETER :: sc_factor=1.  ! to scale up the wildfire emissions, TBD later
+   real(kind_phys), INTENT(IN) :: sc_factor  ! to scale up the wildfire emissions, Jordan please make this a namelist option
```

Comment:
You can edit out the comment to Jordan since this was done in https://github.com/NOAA-EMC/fv3atm/pull/841